src/main/java/com/aws/greengrass/lifecyclemanager/LogManagerHelper.java

```diff
@@ -28,7 +27,6 @@ private LogManagerHelper() {
      * @return a logger with configuration to log to a log file with the same name.
      */
     public static Logger getComponentLogger(GreengrassService service) {
-        // TODO: [P41214167]: Dynamically reconfigure service loggers
```

Comment:
can't remove this. still needed. this means we could set logger config at component level

Reply:
Oh when I open the SIM it says "On logger configuration change, update all service loggers". Thought that's already what we're doing. I'll add "individually" here.